xbmc/LockState.h

```diff
@@ -0,0 +1,16 @@
+/*
+ *  Copyright (C) 2005-2020 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+typedef enum
+{
+  LOCK_STATE_NO_LOCK           =  0,
+  LOCK_STATE_LOCK_BUT_UNLOCKED =  1,
```

Comment:
where is this used?

Reply:
actually only LOCK_STATE_LOCKED is used in this PR (d7862eb), but not a must have